fireREST/fmc/__init__.py

```diff
@@ -269,9 +269,13 @@ def refresh(self):
             logger.info('Access token is invalid. Refreshing authentication token')
             self.refresh_counter += 1
             url = f'{self.protocol}://{self.hostname}{defaults.API_REFRESH_URL}'
-            response = self._request('post', url)
-            self.headers['X-auth-access-token'] = response.headers['X-auth-access-token']
-            self.headers['X-auth-refresh-token'] = response.headers['X-auth-refresh-token']
+            try:
+                response = self._request('post', url)
+                self.headers['X-auth-access-token'] = response.headers['X-auth-access-token']
+                self.headers['X-auth-refresh-token'] = response.headers['X-auth-refresh-token']
+            except exc.GenericApiError as e:
```

Comment:
```suggestion
            except exc.GenericApiError:
```
`e` is not used, so I think we can skip it.

Reply:
You are 100% right. That's an oopsie on my side o.O